cli/src/main.rs

```diff
@@ -62,6 +63,25 @@ fn main() {
                     params.validation_flags =
                         naga::valid::ValidationFlags::from_bits(value).unwrap();
                 }
+                "index-bounds-check-policy" => {
+                    let value = args.next().unwrap();
+                    params.index_bounds_check_policy = match value.as_str() {
+                        "Restrict" => naga::back::IndexBoundsCheckPolicy::Restrict,
+                        "ReadZeroSkipWrite" => {
+                            naga::back::IndexBoundsCheckPolicy::ReadZeroSkipWrite
+                        }
+                        "UndefinedBehavior" => {
+                            naga::back::IndexBoundsCheckPolicy::UndefinedBehavior
+                        }
+                        other => {
+                            panic!(
+                                "Unrecognized '--index-bounds-check-policy' value: {:?}",
+                                other
+                            );
+                        }
+                    };
+                }
+                #[cfg(feature = "spv-in")]
```

Comment:
Accidentally added feature gate?

Reply:
Oh dear. I'd better look over the rest of this PR more thoroughly. Thank you!